colabs/preprocessing.ipynb

```diff
@@ -0,0 +1,253 @@
+{
```

Comment:
Te rzeczy bym przeniósł do katalogu `code examples/colabs`. To w końcu nie są 'nasze' ładowania danych